apps/prairielearn/python/test/core_test.py

```diff
@@ -502,6 +502,10 @@ def test_string_from_numpy(value: Any, args: dict, expected_output: str) -> None
         (np.complex64(complex(1, 2)), {}, "1.0+2.0j"),
         (np.complex64(complex(0, 2)), {}, "0.0+2.0j"),
         (np.complex64(complex(1, 0)), {}, "1.0+0.0j"),
+        # For legacy reasons, we must also support strings.
+        ("0", {}, "0.0"),
+        ("0", {"digits": 1}, "0."),
+        ("0", {"digits": 0}, "0"),
```

Comment:
Maybe add a short test that isn't just zero (like with a complex number).

Reply:
I don't think we ever did or could support complex numbers as strings:

```py
>>> import numpy as np
>>> np.iscomplexobj(str(complex(1,2)))
False
```

I can add a test for a non-zero number in a fast-follow PR, I really just want to land this one so I can start deploying and get this out today. Does that sound alright?

Reply:
Ahh makes sense, sounds good 👍